src/targets/cpu/dnnl.cpp

```diff
@@ -2,6 +2,9 @@
 
 #if defined(__GNUC__) && __GNUC__ <= 5
 namespace std {
+#ifdef MIGRAPHX_ENABLE_ZENDNN
+namespace dnnl = zendnn;
```

Comment:
Wouldn't `dnnl=zendnn` assignment come from `dnnl.hpp` ? If so, do we need another assignment here ?

Reply:
I am getting compiler errors with this removed.

Reply:
If errors are not easily fixable then, it is okay, we can keep this.

Reply:
dnnl.hpp does not do `dnnl=zendnn` as it is completely unaware of zendnn. Also zendnn does not provide such an alias either, so this is necessary.